README.md

```diff
@@ -1,11 +1,63 @@
 ## `service-catalog`
 
-The service-catalog presents APIs for:
-
-1.  Determining which services and recipes for running instances of services
-    are available in a cluster
-2.  Making those services usable in namespaces and more granular subjects
-    (deployments, pods) within a Kubernetes cluster
+### Terminology
+
+Application: code that will access a Service. While in Kubernetes the code that
```

Comment:
I would generalize rather than specify right now and address naming in subsequent PRs to add specific API resources.  Do you agree that leaving as 'Service' for now at least clarifies the terminology in the existing PR?